rita/src/rita_common/dao_manager/mod.rs

```diff
@@ -148,12 +153,29 @@ fn timer_check(timestamp: Instant) -> bool {
     }
 }
 
+/// Sends off a message to TunnelManager about the dao state
+fn send_membership_message(on_dao: bool, their_id: Identity) -> () {
```

Comment:
I'd simplify it TBH - instead of `bool` arg I'd accept `TunnelAction` directly. This way we could save few branches of code.

Reply:
I think this is better since it encapsulates the TunnelManager state stuff, but I'm pretty neutral overall.